packages/service-library/src/servicelib/fastapi/tracing.py

```diff
@@ -16,6 +16,40 @@
 from settings_library.tracing import TracingSettings
 
 log = logging.getLogger(__name__)
```

Comment:
nowadays we do

```suggestion
_logger = logging.getLogger(__name__)
```

since this is a private variable (ongoing evolving best practices)

Reply:
ok thx done